hw18/Lesson18.py

```diff
@@ -0,0 +1,142 @@
+# create class Bot
+class Bot:
+    def __init__(self, name):
+        self.name = name
+
+    def say_name(self):
+        print(self.name)
+
+    def send_message(self, message):
+        print(message)
+
+
+some_bot = Bot('Marvin')
+some_bot.say_name()
+some_bot.send_message('Hello guys')
+
+# Create class TelegramBot inherited from Bot
+
+
+class TelegramBot(Bot):
+
+    def __init__(self, name, url=None, chat_id=None):
+        self.url = url
+        self.chat_id = chat_id
+        name = super().__init__(name) # Which of those 2 rows should I use?
+        # Bot.__init__(self, name)    # Which of those 2 rows should I use?
```

Comment:
```suggestion
        super().__init__(name)
```
it will call `__init__` method of `Bot` and set `name` there